lib/settings-panel.coffee

```diff
@@ -120,7 +120,7 @@ class SettingsPanel extends CollapsibleSectionPanel
 
   set: (name, value) ->
     if @options.scopeName
-      if value is undefined
+      if value is undefined or `value == atom.config.get(name)`
```

Comment:
First of all, we don't allow the use of "embedded JavaScript", so you'll have to drop the backticks here regardless of justification. Which brings me to the second part; why are you using backticks here? Are you actively trying to avoid `===`?